src/unbound.py

```
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+#
+# igcollect - Stats for Unbound DNS cache/resolver
+#
+# Copyright (c) 2016 InnoGames GmbH
+#
+
+from __future__ import print_function
+from argparse import ArgumentParser
+from subprocess import Popen, PIPE
+from time import time
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='unbound')
+    return parser.parse_args()
+
+
+def parse_unbound_stats():
+    out = Popen(('/usr/sbin/unbound-control', 'stats'), stdout=PIPE).\
```

This is silently skipping errors.  Why don't you just use `check_output()`?